src/BinaryParsers/PEBinary/ProgramDatabase/ObjectModuleDetails.cs

```diff
@@ -224,28 +224,35 @@ public string GetDialect(out string versionNumber)
         }
     }
 
+    /// <summary>
+    /// Information about the CV_CFL_LANG enumeration, 
+    /// which specifies the code language of the application or linked module in the debug interface access SDK.
+    /// https://docs.microsoft.com/en-us/visualstudio/debugger/debug-interface-access/cv-cfl-lang
+    /// </summary>
     public enum Language : uint
     {
         C = 0x00,
         Cxx = 0x01,
         FORTRAN = 0x02,
-        MASM = 0x03,
+        MASM = 0x03, // Microsoft Macro Assembler
         Pascal = 0x04,
         Basic = 0x05,
         COBOL = 0x06,
-        LINK = 0x07,
-        CVTRES = 0x08,
-        CVTPGD = 0x09,
+        LINK = 0x07, // linker-generated module
+        CVTRES = 0x08, // resource module converted with CVTRES tool
```

Comment:
>resource [](http://example.com/codeflow?start=26&length=8)

All comments need to start with an upper cased word.

Please prefer adding periods to the end of sentences.